src/hid/hid-device.cpp

```diff
@@ -273,6 +313,12 @@ namespace librealsense
             if(fps > 0)
                 featureReport.report = (1000 / fps);
 
+            //we want to change the sensitivity values only in gyro, for FW version>=5.16
+            if( (int)featureReport.reportId == REPORT_ID_GYROMETER_3D
+                && _realsense_hid_report_actual_size == sizeof( FEATURE_REPORT ) )
```

Comment:
`sizeof( FEATURE_REPORT )` is 9. Are you sure this is the correct struct to use?

Reply:
Fixed